docs/snippets/angular/my-component-story-use-globaltype-backwards-compat.ts.mdx

```diff
@@ -1,10 +1,12 @@
 ```ts
 // MyComponent.stories.ts
 
-export const StoryWithLocale = ({ globals: { locale } }) => {
-  const caption = getCaptionForLocale(locale);
-  return {
-    template: `<p>${caption}</p>`,
-  };
+export const StoryWithLocale = {
+  render: ({ globals: { locale } }) => {
```

Comment:
```suggestion
  render: (args, { globals: { locale } }) => {
```

Reply:
I don't think we have a back-compat version of the render function

Reply:
I'll revert back this snippet to its original form to be safe. Can update it later if it's actually implemented.

Reply:
I've reverted back to what's currently documented with the latest commit. Marking this as solved.